airbyte-webapp/src/components/ui/Link/Link.module.scss

```diff
@@ -0,0 +1,15 @@
+@use "scss/colors";
+
+.link {
+  color: colors.$grey-300;
+  text-underline-offset: 2px;
+
+  &--primary {
+    color: colors.$blue-400;
+    text-decoration: none;
+
+    &:hover {
```

Comment:
Wouldn't we want any hover effect for normal links as well? I think we could make them slightly darker on hover?

Reply:
I agree, but since @Upmitt is working on standardizing text colors & inline link styling, I would prefer to wait for that before changing the behavior. At the moment, most of our "default" light gray links don't have hover effects.